src/App.js

```diff
@@ -60,7 +62,9 @@ export default function App() {
             <ErrorBoundary>
               <Suspense fallback={<SuspenseFallback />}>
                 <Switch>
-                  <Route exact path="/" component={HomePage} />
+                  {/* <Route exact path="/" component={HomePage} /> */}
```

Comment:
Consider removing the comment here, once the experiment is over we will probably just rename the component